DonateSpareChange/qt.py

```diff
@@ -882,9 +882,11 @@ def get_data(self):
                     (True, "eatBCH", "pp8skudq3x5hzw8ew7vzsw8tn4k8wxsqsv0lt0mf3g"),
                     (True, "eatBCH_SS", "qrsrvtc95gg8rrag7dge3jlnfs4j9pe0ugrmeml950"),
                     (True, "Coins4Clothes", "qzx4tqcldmvs4up9mewkf3ru0z6vy9wm6qm782fwla"),
+                    (True, "Cashshuffle", "qqqxxmjyavdkwdj6npa5w6xl0fzq3wc5furaqdpl59"),
```

Comment:
Can you push another commit if you got time? Official name is "CashShuffle" , not "Cashshuffle" (we should try and make it consistent I think given what the logo looks like).

Reply:
I always fuck that up smh